fix: add aria-label for links without text

ojeytonwilliams opened the pull request:
This replaces the alt text, since that becomes redundant

ahmadabdolsaheb commented:
Thanks for the PR. There are a couple of conflicts. could you please take a look.

ojeytonwilliams commented:
Oh, I thought I was being clever by building this from the other PR.  Seems that doesn't work!

ojeytonwilliams commented:
@ahmadabdolsaheb could you take another look?  I'd like to get this in, so we can use the lts version of Node and put that in the docs.

ojeytonwilliams committed:
fix: add aria-label for links without text

This replaces the alt text, since that becomes redundant

ojeytonwilliams committed:
fix: add back alt text

While Lighthouse doesn't care either way, the WAVE AIM tool
considers missing alt text to be an error when a link only contains an
image

naomi-lgbt reviewed (approved):
Looks good to me. Resolves another Lighthouse opportunity.
